apps/frontend/src/app/3_organisms/FundingHistoryFrame/FundingHistoryFrame.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { nanoid } from 'nanoid';
+import { useTranslation } from 'react-i18next';
+
+import {
+  NotificationType,
+  OrderDirection,
+  OrderOptions,
+  Pagination,
+  Table,
+  applyDataAttr,
+} from '@sovryn/ui';
+
+import { ExportCSV } from '../../2_molecules/ExportCSV/ExportCSV';
+import { useNotificationContext } from '../../../contexts/NotificationContext';
+import { useAccount } from '../../../hooks/useAccount';
+import { useBlockNumber } from '../../../hooks/useBlockNumber';
+import { translations } from '../../../locales/i18n';
+import { rskClient } from '../../../utils/clients';
+import {
+  DEFAULT_HISTORY_FRAME_PAGE_SIZE,
+  EXPORT_RECORD_LIMIT,
+} from '../../../utils/constants';
+import {
+  BitcoinTransfer,
+  BitcoinTransferDirection,
+  BitcoinTransfer_OrderBy,
+  useGetFundingLazyQuery,
+} from '../../../utils/graphql/rsk/generated';
+import { useGetFundingHistory } from './hooks/useGetFundingHistory';
+import { FundingHistoryType } from './types';
+import { columnsConfig, generateRowTitle } from './utils';
+
+const pageSize = DEFAULT_HISTORY_FRAME_PAGE_SIZE;
+
+export const FundingHistoryFrame: FC = () => {
+  const { t } = useTranslation();
+  const { account } = useAccount();
+  const { addNotification } = useNotificationContext();
+  const [fundingHistory, setFundingHistory] = useState<FundingHistoryType[]>(
+    [],
+  );
+  const { value: block } = useBlockNumber();
+
+  const [page, setPage] = useState(0);
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'updatedAtTimestamp',
+    orderDirection: OrderDirection.Desc,
+  });
+
+  const { data, loading, refetch } = useGetFundingHistory(
+    account,
+    pageSize,
+    page,
+    orderOptions,
+  );
+
+  useEffect(() => {
+    refetch();
+  }, [refetch, block]);
+
+  const [getFundingHistory] = useGetFundingLazyQuery({
+    client: rskClient,
+  });
+
+  const funding = useMemo(
+    () => (data?.bitcoinTransfers as BitcoinTransfer[]) || [],
+    [data],
+  );
+
+  useEffect(() => {
+    if (account && funding) {
+      //split each row returned from the graph into 2 rows (part 1 and part 2)
+      const data: FundingHistoryType[] = funding.reduce(
+        (acc: FundingHistoryType[], item) => {
+          const row1 = {
+            timestamp: item.createdAtTimestamp,
+            type: BitcoinTransferDirection.Incoming,
+            sent: item.totalAmountBTC,
+            received: null,
+            serviceFee: null,
+            txHash: item.bitcoinTxHash,
```

Comment:
The BTC transaction links are currently broken since it's using RSK explorer for everything - they should use URL from `apps\frontend\src\utils\constants.ts::btcExplorer` instead

Reply:
updated with  getBtcExplorerUrl, getRskExplorerUrl